cockatrice/src/sequenceEdit/secuenceedit.cpp

```diff
@@ -0,0 +1,150 @@
+#include "secuenceedit.h"
+#include "../settingscache.h"
+#include <QLineEdit>
+#include <QPushButton>
+#include <QHBoxLayout>
+#include <QEvent>
+#include <QKeyEvent>
+#include <QToolTip>
+
+SecuenceEdit::SecuenceEdit(QString _shorcutName, QWidget *parent) : QWidget(parent)
+{
+    shorcutName = _shorcutName;
+    currentKey = 0;
+    maxKeys = 4;
+    keys = 0;
+    valid = false;
+
+    lineEdit = new QLineEdit(this);
+    clearButton = new QPushButton("", this);
+    defaultButton = new QPushButton("", this);
+
+    lineEdit->setMinimumWidth(100);
+    clearButton->setMaximumWidth(lineEdit->height());
+    defaultButton->setMaximumWidth(lineEdit->height());
```

Comment:
These two lines cause a problem at least under osx; the buttons are overlapping:
![schermata 2015-08-18 alle 22 46 08](https://cloud.githubusercontent.com/assets/1631111/9342288/2cdca182-45fb-11e5-92c7-ec8ce0a0e465.png)
Commenting out these lines fixes the problem:
![schermata 2015-08-18 alle 22 47 00](https://cloud.githubusercontent.com/assets/1631111/9342300/3bb9138e-45fb-11e5-94bd-195c8e0e105e.png)